notebooks/cosine_io.ipynb

```diff
@@ -0,0 +1,1789 @@
+{
```

Comment:
<p>Because of the instability of this feature, and in light of Jan's concerns from the meeting today, it may be nice to remove this entire job-loading section, or at least shift it all the way to the very end of the tutorial as an "in-development" section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron_contrib/pull/632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
I removed the section for now. It right now is about getting data up!